tests/sles4sap/saptune/mr_test.pm

```diff
@@ -48,10 +48,10 @@ sub setup {
     # Disable packagekit
     quit_packagekit;
     # Install saptune
-    zypper_call "-n in saptune";
+    zypper_call "in saptune";
+    zypper_call "in sapconf";
     if (systemctl("-q is-active sapconf.service", ignore_failure => 1)) {
```

Comment:
Are you certain this check will continue to work even with `sapconf` being installed in the line before? Couldn't confirm that with the verification run as it doesn't seem to have the call to `zypper_call "in sapconf"`

Reply:
Yes, I am very sure from my past experiences. For "12sp4+" the "sapconf" is already installed by default. Reinstall it for all OS versions is OK/simple.
Here is one of the OSD auto runs on latest build (15sp5 build 88.1): https://openqa.suse.de/tests/10847361#step/mr_test/23

Reply:
Good pointing, sorry I had pasted the cloned job link to the VR.
I had updated the VR link to: https://openqa.nue.suse.com/tests/10844459#step/mr_test/21